test/e2e/client_tls_test.go

```diff
@@ -889,8 +876,8 @@ func TestMTLSWithCRLs(t *testing.T) {
 				}
 				return false, nil
 			})
-			// Create CA cert bundle.
-			clientCAConfigmapName := "client-ca-cm-" + namespace.Name
+			// Create CA cert bundle
```

Comment:
Please keep the punctuation.  (This comment could also be a `t.Log` instead, but we can leave that change for another day.)

Reply:
Fixed